File: SqlOrganize/Model/EntityTree.cs

namespace SqlOrganize.Model
{
    /*
    Lectura de json
    */
    public class EntityTree
    {
        public string fieldId { get; set; }
        public string entityName { get; set; }
        public string fieldName { get; set; }
        public string refEntityName { get; set; }
        public string refFieldName { get; set; }
        public Dictionary<string, EntityTree> children { get; set; }

    }
}

File: SqlOrganize/Model/EntityTree.php
<?php

namespace SqlOrganize\Model;

/**
 * Lectura de json
 */
class EntityTree
{
    public string $fieldId;
    public string $entityName;
    public string $fieldName;
    public string $refEntityName;
    public string $refFieldName;
    
    /**
     * @var array<string, EntityTree> Diccionario de hijos
     */
    public array $children = [];
    
    public function __construct()
    {
        $this->children = [];
    }
}